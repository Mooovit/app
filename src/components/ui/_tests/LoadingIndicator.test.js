import React from "react";
import renderer from "react-test-renderer";

const FakeComponent = () => <>Hello world</>

it("rend correctly", () => {
  const tree = renderer.create(<FakeComponent />).toJSON();

  expect(tree).toMatchSnapshot();
});
