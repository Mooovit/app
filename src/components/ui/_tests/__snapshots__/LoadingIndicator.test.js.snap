// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rend correctly 1`] = `"Hello world"`;
